manifests/g/Geeks3D/FurMark/1.32.1.0/Geeks3D.FurMark.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Geeks3D.FurMark
+PackageVersion: 1.32.1.0
+InstallerLocale: en-US
+InstallerType: exe
+Scope: machine
+InstallModes:
+- silentWithProgress
+InstallerSwitches:
+  Silent: /Silent
+  SilentWithProgress: /Silent
+UpgradeBehavior: install
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://datanet.pt/mirror/Geeks3D/FurMark/1.32.1.0/FurMark_1.32.1.0_Setup.exe
```

Comment:
```suggestion
  InstallerUrl: https://geeks3d.com/downloads/2022p/FurMark_1.32.1.0_Setup.exe
```

Reply:
Hi @stephengillie,
That download url does not work if the referrer is not Geeks3D, they protected it from direct linking and that is why FurMark package is not yet available on winget.
What I did is I contacted JegX the maintainer of the site and offered to provide a mirror website for downloads (not needed to be advertised on furmark website) where he can upload the setup binaries when new versions are released.
I have written confirmation from him, do you think anything more is needed to use https://datanet.pt/mirror ?

Reply:
The described scenario does not sound like it has the full agreement of the publisher. If the publisher's security policies prevent downloads except for directly from their site, then the package might not be a good choice for inclusion into Winget. From our [submission expectations](https://learn.microsoft.com/en-us/windows/package-manager/package/repository#validation-process):
> The installer comes directly from the publisher's website.

Reply:
By what I already seen from some packages manifesta, some use a mirror location and not the publisher's website.
I can share with you the written consent I got from Geeks3D.
If nota, ok, se can cancel this PR.

Reply:
Some publishers do split their package hosting off from their other content hosting, and have a download link on their site that either links directly to the other host, or redirects to the other host. This is something that I frequently manually verify when the Validation-Domain label is added: if I goto the PackageUrl or the publisher's main page, and navigate to the installer, does it match the InstallerUrl? 

If Geeks3D controlled the hosting, and provided this on their site as a separate download link, it could be accepted. Or if Geeks3D could allow Winget downloads through their security systems, that would also work.